libs/ui/.claude/CLAUDE.md

```
@@ -0,0 +1,107 @@
+
+Before creating files, read the relevant guide:
+
+| Task | Guide | When |
+|------|-------|------|
+| Component (.tsx) | [components.md](./instructions/components.md) | Always for new components |
+| Tokens (.css) | [tokens.md](./instructions/tokens.md) | When creating/modifying tokens |
+| Story (.stories.tsx) | [storybook.md](./instructions/storybook.md) | Always for new stories |
+
+## Decision Tree
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)